<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Find</class>
 <widget class="QDialog" name="Find">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>128</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Findâ€¦</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="find_button">
     <property name="text">
      <string>Find next</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="cancel_button">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="checks_and_radios_layout">
     <item>
      <layout class="QVBoxLayout" name="checkbox_layout">
       <item>
        <widget class="QCheckBox" name="match_case_checkbox">
         <property name="text">
          <string>Match case</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="wrap_around_checkbox">
         <property name="text">
          <string>Wrap around</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Direction</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="up_radio">
          <property name="text">
           <string>Up</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="down_radio">
          <property name="text">
           <string>Down</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="find_text_layout">
     <item>
      <widget class="QLabel" name="find_label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Find:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="find_line_edit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
